### ISSUE TITLE
enabledManagers gets overwritten or is broken

### ISSUE THREAD

**Issue by moehlone** [renovatebot/renovate#7030]
**What Renovate type are you using?**

Type: self-hosted
Platform: Bitbucket
Version: 22.21.1

**Describe the bug**

I specified `enabledManagers: ["npm"]` within my `renovate.json`, but renovate creates PRs for all dependency types (in my case Docker, Maven, NPM). It works when I pass `--enabled-managers npm` as CLI argument.

**Relevant debug logs**

**To Reproduce**

Use this renovate.json

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["default:base"],
  "enabled": true,
  "platform": "bitbucket",
  "gitAuthor": "Renovate Bot <bot@renovateapp.com>",
  "username": "renovate-bot",
  "enabledManagers": ["npm"],
  "schedule": ["after 9am every weekday", "before 5pm every weekday"],
  "automerge": true,
  "major": {
    "automerge": false
  },
  "vulnerabilityAlerts": {
    "labels": ["security"],
    "enabled": true
  }
}
```

Run
`renovate --platform=bitbucket --password=$RENOVATE_APP_PASSWORD --username=$RENOVATE_USER ***/***`
and renovate will match all dependency types.

Run
`renovate --platform=bitbucket --password=$RENOVATE_APP_PASSWORD --username=$RENOVATE_USER --enabled-managers npm ***/***`
and renovate will only match npm dependency types.

**Additional context**

**Comment by viceice** [comment]
regression of v22.21.0 and should be fixed by #7028 and v22.21.2

**Comment by moehlone** [comment]
Thanks, I have just realized that in my case this also applies to automerge.. I am too slow for your release cycles :D 
It works now as expected.
